docs/copied-from-beats/security/linux-seccomp.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[linux-seccomp]]
```

Comment:
#928 has been open for that, I can pick it up before merging this though I don't expect any issues.

Reply:
This has been enabled by default since it was merged - libbeat already logs installation status on startup - 

```
{
  "level":"info",
  "logger":"seccomp",
  "caller":"seccomp/seccomp.go:116",
  "message":"Syscall filter successfully installed"
}
```

I tested enable/disable and the `kill_process ` action - nothing was triggered when hitting the various apm-server endpoints, as expected.  #928 can be closed once this pull request is merged.

Reply:
great, thanks for digging in  👍